lib/writer.js

```diff
@@ -33,6 +33,10 @@ exports.getCommitUrl = function (baseUrl, commitHash) {
     urlCommitName = 'commits';
   }
 
+  if (baseUrl.indexOf('gitlab') > -1 && baseUrl.slice(-4) === '.git') {
```

Comment:
Nit: For parity with the `bitbucket` check, can you make the index check `!== -1` instead of `> -1`?

Reply:
sure, I will do it.